cylc/flow/network/resolvers.py

```diff
@@ -639,23 +639,29 @@ async def nodes_mutator(self, *m_args):
                 args['task_job'] = items[0]
             else:
                 args['tasks'] = items
-        result = await self._mutation_mapper(command, args)
+        result = await self._mutation_mapper(command, args, meta)
         if result is None:
             result = (True, 'Command queued')
         return [{'id': w_id, 'response': result}]
 
     async def _mutation_mapper(
-        self, command: str, kwargs: Dict[str, Any]
+        self, command: str, kwargs: Dict[str, Any], meta: Dict[str, Any]
     ) -> Optional[Tuple[bool, str]]:
         """Map between GraphQL resolvers and internal command interface."""
         method = getattr(self, command, None)
```

Comment:
I think this is the first point at which the GraphQL routing has been finalised so in theory we can move the `client-command` logging down here, raise the authorised user logging up here and merge the two together to avoid having two separate log messages for each mutation.

If we do that I don't think we need to pass meta any further down the chain?

WDYT?

Reply:
👍 , I'll give that a go.